code/registration.py

```diff
@@ -1416,33 +1579,28 @@ def multiplane_motion_correction(data_dir: Path, output_dir: Path, debug: bool =
     frame_rate_hz: float
         frame rate in Hz
     """
-    try:
-        unique_id = [i for i in data_dir.rglob("*") if "ophys_experiment" in str(i)][
-            0
-        ].name.split("_")[-1]
-        h5_file = [i for i in data_dir.rglob("*") if f"{unique_id}.h5" in str(i)][0]
-    except IndexError:
-
-        unique_id = [i for i in data_dir.rglob("*") if i.is_dir()][0].name
-        h5_file = [i for i in data_dir.rglob("*") if f"{unique_id}.h5" in str(i)][0]
-    session_dir = h5_file.parent.parent
-    platform_json = next(session_dir.glob("*platform.json"))
-    # this file is required for paired plane registration but not for single plane
-    # in the future, we should make this file accessible to the pipeline through channel connections
-    # instead of needing to copy it from here
-    with open(platform_json, "r") as j:
-        platform_data = json.load(j)
+    h5_dir = [i for i in data_dir.rglob("*VI*") if i.is_dir()][0]
```

Comment:
I don't think "VI" will always be a substring of the directory name!

Reply:
Good point @j-friedrich . Could this instead get all folder names within /pophys/ ?